### PR TITLE
RDAPI-30769 APIM Release notes updates for GraphQL

### PR DESCRIPTION
**Pull request by lcattolico**
http://pr-2757.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/apim_relnotes/20231130_apimgr_relnotes/

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">âœ…</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">ðŸ”¨</span> Latest commit | ef70d0a2 |
 |<span aria-hidden="true">ðŸ˜Ž</span> Deploy preview | http://PR-2757.axway-open-docs.opendocs-builder.pcloud.axway.int |

File: content/en/docs/apim_relnotes/20231130_apimgr_relnotes.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ placeholder
 
 It is important, especially when upgrading from an earlier version, to be aware of the following changes in the behavior or operation of the product in this update, which might impact your current installation.
 
-### placeholder 2
+### GraphQL support
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20231130_apimgr_relnotes.md:39]: Support for GraphQL APIs

Comment (lcattolico) [reply content/en/docs/apim_relnotes/20231130_apimgr_relnotes.md]: Done

**Review by jmbermingham — APPROVED** [review]
Yep, looks good.

File: content/en/docs/apim_relnotes/20231130_apimgr_relnotes.md [hunk 2]
```diff
@@ -36,9 +36,17 @@ placeholder
 
 It is important, especially when upgrading from an earlier version, to be aware of the following changes in the behavior or operation of the product in this update, which might impact your current installation.
 
-### placeholder 2
+### Support for GraphQL APIs
 
-placeholder
+This update of API Gateway ships with support for GraphQL APIs. This enables customers to import GraphQL specifications and apply security and policies, as well as view the specification in API Manager and try it out.
+
+Now, from API Manager UI, you can import (register) a GraphQL back-end API by way of the API defition file, URL, or URL Introspection. Following, you can virtualize your registered back-end API as a publicly exposed front-end API, then secure it and manage the API lifecycle.
```
[the remark this answers is not in the document]
Comment (dshielaxway) [reply content/en/docs/apim_relnotes/20231130_apimgr_relnotes.md:43]: I think I'd rephrase the 2nd sentence slightly too. How about,

Subsequently, you can virtualize your registered back-end API exposing it as a publicly accessible front-end API, secure it and manage its API lifecycle.

**Review by dshielaxway — CHANGES_REQUESTED** [review]

File: content/en/docs/apim_relnotes/20231130_apimgr_relnotes.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jmbermingham) [reply content/en/docs/apim_relnotes/20231130_apimgr_relnotes.md:43]: That change looks fine, I think its better.

[the remark this answers is not in the document]
Comment (andreamussap) [reply content/en/docs/apim_relnotes/20231130_apimgr_relnotes.md:43]: @dshielaxway what about this one, do I keep it? Is "Alternatively" correct?

"Alternatively, API Manager allows you to proxy GraphQL APIs."

[the remark this answers is not in the document]
Comment (dshielaxway) [reply content/en/docs/apim_relnotes/20231130_apimgr_relnotes.md:43]: Sorry, I didn't answer the question.

I'd leave it as 'Subsequently, ...'

File: content/en/docs/apim_relnotes/20231130_apimgr_relnotes.md [hunk 3]
```diff
@@ -36,9 +36,13 @@ placeholder
 
 It is important, especially when upgrading from an earlier version, to be aware of the following changes in the behavior or operation of the product in this update, which might impact your current installation.
 
-### placeholder 2
+### Support for GraphQL APIs
 
-placeholder
+This update of API Gateway ships with support for GraphQL APIs. This enables customers to import GraphQL specifications, virtualize the registered back-end API exposing it as a publicly accessible front-end API, secure it, and manage its API lifecycle. You can also view the specification in API Manager Catalog, and try it out.
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20231130_apimgr_relnotes.md:41]: @dshielaxway I had to make some change in order to use your suggestion. There was no need to use "subsequently" after that. Please let me know if the text is ok. Thanks.

Comment (dshielaxway) [reply content/en/docs/apim_relnotes/20231130_apimgr_relnotes.md:41]: Yeah, that reads better. Thanks!
